src/global.ts

```diff
@@ -39,24 +44,41 @@ export interface BaseLoginOptions {
    * This currently only affects the classic Lock experience.
    */
   login_hint?: string;
+
   acr_values?: string;
+
   /**
    * The default scope to be used on authentication requests.
    * The defaultScope defined in the Auth0Client is included
    * along with this scope
    */
   scope?: string;
+  
   /**
    * The default audience to be used for requesting API access.
    */
   audience?: string;
+  
   /**
    * The name of the connection configured for your application.
    * If null, it will redirect to the Auth0 Login Page and show
    * the Login Widget.
    */
   connection?: string;
-
+  
+  /**
+   * Log users in to a specific organization (Organizations is currently a Closed Beta).
+   *
+   * This will specify an `organization` parameter in your user's login request and will add a step to validate
+   * the `org_id` claim in your user's ID Token.
+   */
+  organization?: string;
```

Comment:
```suggestion
  /**
   * The Id of an organization to which the user should log in to (Organizations is currently a Closed Beta).
   *
   * This will specify an `organization` parameter in your user's login request and will add a step to validate
   * the `org_id` claim in your user's ID Token.
   */
  organization?: string;
```

Reply:
Agree with that in principal, the wording is a little verbose. Let's go with:

> The Id of an organization to log in to..

What do you think?

Reply:
Updated the suggestion!